precompiles/xcm/XCM_v2.sol

```diff
@@ -0,0 +1,80 @@
+pragma solidity ^0.8.0;
+
+/**
+ * @title XCM interface.
+ */
+interface XCM {
+    // A multilocation is defined by its number of parents and the encoded junctions (interior)
+    struct Multilocation {
+        uint8 parents;
+        bytes[] interior;
+    }
+    
+    /**
+     * @dev Withdraw assets using PalletXCM call.
```

Comment:
You shouldn't put implementation details into interface definition.

I see the old file also has this, but it's wrong.